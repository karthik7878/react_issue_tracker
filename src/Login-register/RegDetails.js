import React from 'react';
import { Link } from 'react-router-dom';
export default class RegDetails extends React.Component{

    constructor(props)
    {
        super(props);

 
      this.state={
          printvals:[],
          regemail:''
         
      }
    }
  
   


    render(){
        const { printvals } = this.state;
     
      this.state.regemail=localStorage.getItem("emailid");
        fetch("http://localhost:3001/users?emailid="+this.state.regemail)
            .then(res => res.json())
          
            .then(result => {
              this.setState({
            printvals: result});
            });
        
         
        return(<div>
          <div>
            <h1>Registration Details </h1>
            <br/>
          {printvals.map(printval => (
            <div>
            <p>Email ID : {printval.emailid}</p>
            <p>Password : {printval.password}</p>
            <p>FirstName : {printval.firstname}</p>
            <p>LastName : {printval.lastname}</p>
            <p>Location : {printval.location}</p>
            <p>Mobile Number :{printval.mobile_number}</p>
            </div>
          ))}
      
          </div>
          <div>
            <p><Link to={`/issue2`}><button className="btn btn-dark">Back</button></Link></p>
          </div>
          </div>
        );
    }

}