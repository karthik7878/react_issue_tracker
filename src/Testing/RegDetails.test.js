import RegDetails from "../Login-register/RegDetails";
import { shallow } from 'enzyme';



let wrapper;

beforeEach(() => {
    wrapper = shallow(<RegDetails/>);
})

it("matches snapshot",()=>{

    expect(wrapper).toMatchSnapshot();
})