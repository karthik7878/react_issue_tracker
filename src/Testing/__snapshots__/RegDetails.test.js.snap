// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h1>
      Registration Details 
    </h1>
    <br />
  </div>
  <div>
    <p>
      <Link
        to="/issue2"
      >
        <button
          className="btn btn-dark"
        >
          Back
        </button>
      </Link>
    </p>
  </div>
</div>
`;
